README.rst

```diff
@@ -29,13 +29,206 @@
         :alt: Release Number
         :target: https://github.com/rero/rero-ils/releases/latest
 
-.. image:: https://img.shields.io/github/license/rero/rero-ils.svg
+.. image:: https://img.shields.io/badge/License-AGPL%20v3-blue.svg
         :alt: License
-        :target: https://github.com/rero/rero-ils/blob/master/LICENSE
+        :target: http://www.gnu.org/licenses/agpl-3.0.html
+.. image:: https://img.shields.io/gitter/room/rero/reroils.svg
+        :alt: Gitter room
+        :target: https://gitter.im/rero/reroils
 
-Integrated Library System flavour of Invenio by RERO.
+*Copyright (C) 2020 RERO*
 
-*This is an experimental developer preview release.*
+Table of Content
+----------------
 
-For setup, please read the ``INSTALL.rst`` file.
+1. `Why RERO ILS?`_
+2. `Who’s going to run RERO ILS, in which context?`_
+3. `What does RERO ILS?`_
+4. `How to install RERO ILS?`_
+5. `How to contribute to RERO ILS?`_
+6. `The RERO ILS ecosystem`_
 
+Why RERO ILS?
```

Comment:
I would write "What is RERO ILS", because concretely we give a definition, not *reasons*.